### PR TITLE
Link to Bad metadata Documentation

### LINKED ISSUE #585: Refer (link to) Typechecking Queries from Unit Testing in the doc, regarding check vs checkOutput issue.
It would be helpful to link to [Working Around Bad Metadata](http://tpolecat.github.io/doobie/docs/06-Checking.html) from `Unit Testing` section.

### PR DESCRIPTION
**Pull request by ChristopherDavenport**
As discussed on gitter.

Fixes #585

### CODE REVIEW

**Comment by ChristopherDavenport** [comment]
@tpolecat Just checking in on this to see if I needed to make any changes.

**Comment by tpolecat** [comment]
Yes, sorry, it's in the queue!

**Comment by ChristopherDavenport** [comment]
No worries, just like to make sure I do my due diligence.

File: yax/docs/src/main/tut/13-Unit-Testing.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ As with earlier chapters we set up a `Transactor` and YOLO mode. We will also us
 #+cats
 As with earlier chapters we set up a `Transactor` and YOLO mode. We will also use the `doobie-specs2-cats` and `doobie-scalatest-cats` add-ons.
 #-cats
+If you run into any difficulties it may be useful to refer to the [Working Around Bad Metadata Section](/docs/06-Checking.html)
```
Comment (tpolecat) [inline yax/docs/src/main/tut/13-Unit-Testing.md]: The link should be `06-Checking.html` without the leading `/docs/` … it's confusing, sorry.

Comment (ChristopherDavenport) [reply yax/docs/src/main/tut/13-Unit-Testing.md]: My bad, I'll fix that up.

**Comment by tpolecat** [comment]
Thanks!

**Pull request merged**